src/_data/products.json

```diff
@@ -35,4 +35,27 @@
     "colour": "chartreuse",
     "featured": true
   }
```

Comment:
JSON formatting is what’s failing the deployment, can you add the commas & make sure indentation is consistent?